decentraland/unity-renderer
# feat: highlight username in chat on hover

## What does this PR change?

Underlines the username inside the chat entry when there is a hover over it. This gives the sensation that the button is clickable. Also, it changes the behaviour of how the context menu is opened. Now it opened with a left click instead of a right click. 

## How to test the changes?

1. Open the general chat section. Click on messages that you received, and see that the name becomes underlined. This applies to public and private messages in the general chat. On clicked, the context menu should appear. Clicking anywhere should close it.
2. On the private chat section, when clicking on text messages, the context menu should appear.


## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
